test/functional/crossFunctional.mocha.js

```diff
@@ -404,6 +404,14 @@ describe("Cross Functional", function() {
         .apply('TotalAdded', '$wiki.sum($added)')
     }));
 
+    it('works with length action on filter', equalityTest({
+      executorNames: ['druid', 'mysql', 'postgres'],
+      expression: $('wiki').filter('$cityName.length() > 0')
```

Comment:
this is a very weak test. almost every city name is > 0 maybe do `'$cityName.length() > 10'` seems like it would be a better test